db/institutions.yml

```diff
@@ -355,15 +355,12 @@ kent_state:
   name: Kent State University
   state: OH
   zip: 44242-0001
-caltech:
+caltechu:
```

Comment:
this should be caltech though.  All the "techs" are "institute of Technology"

Reply:
I thought CalTech was CalTech. But MIT is not MTech.